Gruntfile.js

```diff
@@ -19,9 +19,53 @@ module.exports = function (grunt) {
                 },
                 src: ['test/**/*.test.js']
             }
+        },
+        copy: {
```

Comment:
This is the target that copies the sample plugin and converts it to yeoman templates.